PagerDuty/Pipelines/BuildNotifications/README.md

```diff
@@ -43,7 +37,7 @@ An API token is also required for the JFrog Pipelines + Pagerduty Notifications
 1. Open your JFrog Platform with Pipelines and click on the `Adminstration gears` in the left menu. Click on the `Pipelines` menu item and then click on the sub-tab `Integrations`.
 2. Click the `+ Add an Integration` link on the top right.
 3. Select `PagerDuty Events` for the integration type
-![AddNewIntegration](https://github.com/narayanss-jf/partner-integrations/blob/main/PagerDuty/Pipelines/BuildNotifications/AddNewIntegration.png)
+![AddNewIntegration](images/AddNewIntegration.png)
 4. Enter a name for this integration which will be used in the pipeline yaml. ex: `MyPagerDutyIntegration`
 5. Enter the Pagerduty API token created in prior steps.
```

Comment:
should this line also refer to it as "routing key" or "integration key" instead of API token?

Reply:
also it won't let me comment since its not related to this PR, but line 50: 
```
5. Replace `<PagerDutyIntegrationKey>` with the integration key obtained from PagerDuty service UI.
```
i think is no longer needed.